src/libs/ajax/Metrics.ts

```diff
@@ -87,3 +87,4 @@ export const Metrics = (signal?: AbortSignal) => {
     }) as (anonId: string) => Promise<void>,
   };
 };
+export type MetricsAjaxContract = ReturnType<typeof Metrics>;
```

Comment:
```suggestion
export type MetricsContract = ReturnType<typeof Metrics>;
```

Nit. Other Ajax modules follow this naming convention and don't include "Ajax" in the name of the contract type.